docs/cli.md

```diff
@@ -218,6 +219,19 @@ Usage:
 $ s2i rebuild <image name> [<new-tag-name>]
 ```
 
+# s2i generate
+The `s2i generate` command produces a Dockerfile using an existing S2I base 
```

Comment:
Add the text "This is a Tech Preview feature" before this line, and we should be good.

Reply:
Done in https://github.com/openshift/source-to-image/pull/1023/commits/a9c3f7ff062161628a57f1c5948b09a4584e1f5d